daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -151,14 +153,18 @@ damlStart tmpDir = do
     waitForHttpServer
         (threadDelay 100000)
         ("http://localhost:" <> show jsonApiPort <> "/v1/query")
-        authorizationHeaders
+        (authorizationHeaders "Alice") -- dummy party here, not important
```

Comment:
we could instead check `/v1/parties` with an admin token to not have to make up a non-existent party.